index.d.ts

```diff
@@ -2,11 +2,19 @@
 // import strif = require('strif');
 
 export type SupportedLoggers = 'console' | 'file' | 'remote' | 'memory' | 'default';
-export type SupportedSeverities = 'DEFAULT' | 'DEBUG' | 'INFO' | 'NOTICE' | 'WARNING' | 'ERROR' | 'CRITICAL' | 'ALERT' | 'EMERGENCY';
+export type SupportedSeverities =
+  | 'DEFAULT'
+  | 'DEBUG'
+  | 'INFO'
+  | 'NOTICE'
+  | 'WARNING'
+  | 'ERROR'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
  | "ERROR"
```